src/components/Notifications/Notifications.tsx

```diff
@@ -553,6 +555,14 @@ class NotificationEntry extends React.Component<NotificationEntryProps, any> {
                         )}
                     </div>
                 );
+            case "prizeCodeExpiring":
+                return (
+                    <div>
+                        Your {notification.code_level} tier prize code expires in{" "}
```

Comment:
Thanks for adding the notification!

Let's get this wrapped in the `interpolate` and `_` calls to get it translated. Let's tweak the text a little to to include a link to the place where they can sign up if they want, maybe something like:

`Your {code_level} prize code expires in {expires_in_days} days. Click here to visit the supporter page if you'd like to sign up or manage your AI review level."  

Since our translation system doesn't support embedding links nicely, I reckon you can just make that whole second sentence an <a> link to the supporter page and translate the whole sentence within the <a> tag.

Reply:
The notification itself already opens to the supporter page when clicked. Guess we just need to make it clearer in the text of the notification. :)

Reply:
Alright, I added `interpolate` and `_` calls to the notifications and to the redemption page since it didn't have them either.